### PR TITLE
Qt/MainWindow: Remember last directory in "Open" prompt

### PR DESCRIPTION
**Pull request by spycrab**
Fixes [issue #11189](https://bugs.dolphin-emu.org/issues/11189).

### CODE REVIEW

File: Source/Core/DolphinQt2/MainWindow.cpp [hunk 1]
```diff
@@ -844,9 +856,9 @@ void MainWindow::HideRenderWidget(bool reinit)
     setWindowTitle(QString::fromStdString(Common::scm_rev_str));
   }
 
-  // The following code works around a driver bug that would lead to Dolphin crashing when changing
-  // graphics backends (e.g. OpenGL to Vulkan). To avoid this the render widget is (safely)
-  // recreated
```
Comment (leoetlino) [inline Source/Core/DolphinQt2/MainWindow.cpp]: Unrelated change

Comment (spycrab) [reply Source/Core/DolphinQt2/MainWindow.cpp]: the linter forces this change on me though :/
